packages/dataviews/src/utils.ts

```diff
@@ -17,33 +10,14 @@ export function sanitizeOperators< Item extends AnyItem >(
 
 	// Assign default values.
 	if ( ! operators || ! Array.isArray( operators ) ) {
-		operators = [ OPERATOR_IS_ANY, OPERATOR_IS_NONE ];
+		operators = [ 'isAny', 'isNone' ];
 	}
 
-	// Transform legacy in, notIn operators to is, isNot.
-	// To be removed in the future.
-	if ( operators.includes( 'in' ) ) {
-		operators = operators.filter( ( operator ) => operator !== 'is' );
-		operators.push( 'is' );
-	}
-	if ( operators.includes( 'notIn' ) ) {
-		operators = operators.filter( ( operator ) => operator !== 'notIn' );
-		operators.push( 'isNot' );
-	}
-
-	// Make sure only valid operators are used.
-	operators = operators.filter( ( operator ) =>
-		ALL_OPERATORS.includes( operator )
-	);
```

Comment:
I removed this, I'm not sure whether operators can enter this system and potentially be invalid.

If operators are internal, then the check is likely redundant. If not, then a check like this would be good to restore. A type guard like this would work:

```ts
export function isValidOperator( candidate: string ): candidate is Operator {
	switch ( candidate ) {
		case 'is':
		case 'isNot':
		case 'isAny':
		case 'isNone':
		case 'isAll':
		case 'isNotAll':
			return true;
	}
	return false;
}
```

Reply:
They can enter the component and be invalid but it's doing it wrong.